maas2netbox/validators.py

```diff
@@ -14,13 +14,14 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import logging
-from urllib.parse import urlparse
 
 from maas.client.enum import NodeStatus
 
 from maas2netbox import config
 from maas2netbox.utils import ipmi, maas, netbox
 
+from urllib.request import urlparse
```

Comment:
please revert the change of import. It used to be present at line 17

Reply:
Should be OK now